microcloud/test/includes/microcloud.sh

```diff
@@ -442,6 +442,25 @@ reset_snaps() {
 
       lxd waitready
     "
+
+    # Add debug binaries for MicroCloud.
+    if [ -n "${MICROCLOUD_DEBUG_PATH}" ] && [ -n "${MICROCLOUDD_DEBUG_PATH}" ]; then
+      lxc exec "${name}" -- rm -rf /var/snap/microcloud/common/microcloudd.debug
+      lxc exec "${name}" -- rm -rf /var/snap/microcloud/common/microcloud.debug
+
+      lxc file push "${MICROCLOUDD_DEBUG_PATH}" "${name}"/var/snap/microcloud/common/microcloudd.debug
+      lxc file push "${MICROCLOUD_DEBUG_PATH}" "${name}"/var/snap/microcloud/common/microcloud.debug
+
+      lxc exec "${name}" -- sh -c "systemctl restart snap.microcloud.daemon"
```

Comment:
out of interest, why do you use `sh -c` here rather than just calling the command like:

```
lxc exec "${name}" -- systemctl restart snap.microcloud.daemon
```

Reply:
Habit, really. the shorter way is, well, shorter so I'll switch it :)